tests/util/kube_utils.go

```diff
@@ -611,3 +616,70 @@ func GetKubeConfig(filename string) error {
 	log.Infof("kubeconfig file %s created\n", filename)
 	return nil
 }
+
+// CreateMultiClusterSecrets will create the secrets and configmap associated with the remote cluster
+func CreateMultiClusterSecrets(namespace string, KubeClient kubernetes.Interface, RemoteKubeConfig string) error {
+	const (
+		secretName    = "remote-cluster"
+		configMapName = "clusterregistry"
+	)
+	_, err := ShellMuteOutput("kubectl create secret generic %s --from-file %s -n %s", secretName, RemoteKubeConfig, namespace)
+	// The cluster name is derived from the filename used to create the secret we will need it for the configmap
+	filename := filepath.Base(RemoteKubeConfig)
+	if err != nil {
+		return err
+	}
+	log.Infof("Secret remote-cluster created\n")
+	remoteCluster := &v1.ConfigMap{
+		ObjectMeta: meta_v1.ObjectMeta{
+			Name:      configMapName,
+			Namespace: namespace,
+		},
+	}
+
+	remoteClusterData := v1alpha1.Cluster{
+		TypeMeta: meta_v1.TypeMeta{
+			Kind:       "Cluster",
+			APIVersion: "clusterregistry.k8s.io/v1alpha1",
+		},
+		ObjectMeta: meta_v1.ObjectMeta{
+			Name:      secretName,
+			Namespace: namespace,
+			Annotations: map[string]string{"config.istio.io/accessConfigSecret": secretName,
+				"config.istio.io/accessConfigSecretNamespace": namespace,
+				"config.istio.io/pilotEndpoint":               "TODO - will be removed from Pilot",
```

Comment:
alot of these annotations have been removed already - no sense leaving the TODOs in there when a quick delete will do the job.

Reply:
Removed in what PR?  This set was required less than 24 hours ago.